src/audio_player.rs

```diff
@@ -0,0 +1,90 @@
+use crate::sync_wrapped;
+use bevy::prelude::{AudioSinkPlayback, Component, Vec3};
+
+#[derive(Component)]
+pub struct AudioListener {
+    pub previous_position: Vec3,
```

Comment:
What exactly does **previous**_position mean? Is it delayed by a frame?

Reply:
Yes, it's used to calculate the velocity for the 3dAttributes